src/apify_client.ts

```diff
@@ -1,7 +1,7 @@
 import type { SetStatusMessageOptions } from '@crawlee/types';
 import ow from 'ow';
 
-import { ME_USER_NAME_PLACEHOLDER, ACTOR_ENV_VARS } from '@apify/consts';
+import { ACTOR_ENV_VARS,ME_USER_NAME_PLACEHOLDER } from '@apify/consts';
```

Comment:
Duh, the fixer is messing up white spaces, seen that before.

I guess I'll add prettier right ahead instead of dealing with this by hand.

Reply:
Alright, I can see this is in draft, so withholding my review.

Reply:
Hmm, this is dumb, haven't seen it cause problems in any other projects.

Reply:
It happened on crawlee, sdk as well.